contents/staticAnalysis/intro.md

```diff
@@ -0,0 +1,92 @@
+## Static Analysis
+Static analysis is the process of analysing computer programme **without** executing the problem. This practice is often used to ensure the codes follow certain structures or standards (e.g coding standards).
+
+It is possible to do static analysis manually, but there are automated tools(static analysers) that can assist developers in this process. 
+
+### Why Static Analysis?
+
+#### Analyse Thoroughly
```

Comment:
"write all test cases" sounds weird, maybe just write "impossible to achieve 100% test coverage"?